ios/RNSpokestack.m

```diff
@@ -59,11 +59,11 @@ - (void)didRecognize:(SpeechContext * _Nonnull)results {
 
 - (void)activate {
     NSLog(@"RNSpokestack activate");
+    [_pipeline activate];
     if (hasListeners)
     {
         [self sendEventWithName:@"onSpeechEvent" body:@{@"event": @"activate", @"transcript": @"", @"error": @""}];
     }
-    [_pipeline activate];
```

Comment:
Did you mean to change the order here?

Reply:
Yep. The pattern should be to take action then send the event.

Reply:
Oh right, I read this backwards. I thought it was already like that.